packages/react/src/components/Grid/FlexGrid.stories.js

```diff
@@ -289,3 +289,52 @@ export const MixedGridModes = () => (
     </Row>
   </FlexGrid>
 );
+
+export const Playground = (args) => (
+  <FlexGrid {...args}>
+    <Row>
+      <Column>
+        <DemoContent>1/4</DemoContent>
+      </Column>
+      <Column>
+        <DemoContent>1/4</DemoContent>
+      </Column>
+      <Column>
+        <DemoContent>1/4</DemoContent>
+      </Column>
+      <Column>
+        <DemoContent>1/4</DemoContent>
+      </Column>
+    </Row>
+  </FlexGrid>
+);
+
+Playground.argTypes = {
+  as: {
+    control: {
+      type: 'text',
+    },
+    defaultValue: 'div',
+  },
+  fullWidth: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  narrow: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  condensed: {
+    control: {
+      type: 'boolean',
+    },
+    defaultValue: false,
+  },
+  columns: {
+    control: { type: 'number' },
```

Comment:
I'm not seeing any changes when I modify this prop. Do we need to expose this prop?

I think we can also hide `children` and `className` here.

Reply:
Yeah, `columns` should've been removed a long time ago. It was part of the original exploration of CSSGrid. We can remove it separately though (https://github.com/carbon-design-system/carbon/issues/12338), just hide it from the controls pane for now.